lib/baza-rb.rb

```
@@ -696,41 +696,48 @@ def checked(ret, allowed = [200])
+      headers_log = headers.map { |k, v| "  #{k}: #{v}" }.join("\n")
+      @loog.debug("#{log}\n#{headers_log}")
+      @loog.error("#{log}\n#{headers_log}")
+    else
+      @loog.debug("#{log}\n  (no headers returned)")
+      @loog.error("#{log}\n  (no headers returned)")
+    end
+
+    # Compose detailed error message
+    failure_info = []
+    failure_info << "X-Zerocracy-Failure=#{headers['X-Zerocracy-Failure'].inspect}" if headers['X-Zerocracy-Failure']
+    failure_info << "X-Zerocracy-FailureMark=#{headers['X-Zerocracy-FailureMark'].inspect}" if headers['X-Zerocracy-FailureMark']
```

RuboCop (1.81.6)

[convention] Modifier form of `if` makes the line too long.

(Style/IfUnlessModifier)